paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -166,20 +165,6 @@ internal class PaymentOptionsViewModel @Inject constructor(
 
     private fun handleLinkProcessingState(processingState: LinkHandler.ProcessingState) {
         when (processingState) {
-            LinkHandler.ProcessingState.Cancelled -> {
-                onPaymentResult(PaymentResult.Canceled)
-            }
-            is LinkHandler.ProcessingState.PaymentMethodCollected -> {
-                TODO("This can't happen. Will follow up to remodel the states better.")
```

Comment:
Well, I never did this.... This is why we don't add todos. Jira or assume it won't happen.